composer/modules/web/src/plugins/ballerina/diagram/views/default/components/decorators/action-invocation-decorator.jsx

```diff
@@ -144,6 +148,44 @@ class InvocationDecorator extends React.Component {
                                 + designer.config.actionInvocationStatement.timelineHeight;
                 backwardArrowEnd.x = statementBox.x + (designer.config.actionInvocationStatement.width / 2);
                 backwardArrowEnd.y = backwardArrowStart.y;
+
+                if (TreeUtil.isVariableDef(model) ||
+                    TreeUtil.isAssignment(model) ||
+                    TreeUtil.isExpressionStatement(model)) {
+                    let exp;
+
+                    if (TreeUtil.isVariableDef(model)) {
+                        exp = model.getVariable().getInitialExpression();
+                    } else {
+                        exp = model.getExpression();
+                    }
+
+                    if (TreeUtil.isMatchExpression(exp) || TreeUtil.isCheckExpr(exp)) {
+                        exp = exp.getExpression();
+                    }
+
+                    const functionNameWidth = new SizingUtils().getTextWidth(exp.getFunctionName(), 0);
+                    const nodeWidth = viewState.components.text.x -
+                        (designer.config.statement.padding.left + designer.config.statement.padding.right);
+
+                    if (functionNameWidth.w > nodeWidth) {
+                        const truncatedFunctionNameWidth = new SizingUtils().getTextWidth(
+                            exp.getFunctionName(), 0, nodeWidth);
+
+                        expression = (<tspan>
+                            {truncatedFunctionNameWidth.text}
+                        </tspan>);
+                    } else {
+                        const displayExpressionWidth = nodeWidth - functionNameWidth.w;
+                        const expressionDisplayText = new SizingUtils().getTextWidth(model.viewState.parameterText, 0,
+                            displayExpressionWidth).text;
+
+                        expression = (<tspan>
+                            {exp.getFunctionName()}
```

Comment:
did you run the es linter for this ... this line might give you a lint error ??...